.github/workflows/ci.yml

```diff
@@ -7,13 +7,7 @@ on:
 
 jobs:
   build-and-test:
-    runs-on: ${{ matrix.os }}
-    env:
-      USE_BAZEL_VERSION: ${{ matrix.bazel_version }}
-    strategy:
-      matrix:
-        os: [ubuntu-20.04]
-        bazel_version: [bazelbuild/5.4.1, bazelbuild/6.4.0]
```

Comment:
Do we want to use the `bazel-contrib/setup-bazel@0.8.5` step like we do in the other rule sets?

Reply:
Hmm, I'm surprised the builds pass without it. I'll add it.